FTBFS with mutter 3.15.4

Hi,

with mutter 3.15.4 budgie wm fails to build from source:

```
  CC       ui/budgie_wm-background.o
ui/background.c: In function ‘_update’:
ui/background.c:256:17: error: incompatible type for argument 2 of ‘meta_background_set_file’
                 meta_background_set_file(background, style);
                 ^
In file included from /usr/include/mutter/meta/meta-background-actor.h:26:0,
                 from ui/background.c:14:
/usr/include/mutter/meta/meta-background.h:70:6: note: expected ‘struct GFile *’ but argument is of type ‘GDesktopBackgroundStyle’
 void meta_background_set_file     (MetaBackground            *self,
      ^
ui/background.c:256:17: error: too few arguments to function ‘meta_background_set_file’
                 meta_background_set_file(background, style);
                 ^
In file included from /usr/include/mutter/meta/meta-background-actor.h:26:0,
                 from ui/background.c:14:
/usr/include/mutter/meta/meta-background.h:70:6: note: declared here
 void meta_background_set_file     (MetaBackground            *self,
      ^
make[2]: *** [ui/budgie_wm-background.o] Error 1
Makefile:603: recipe for target 'ui/budgie_wm-background.o' failed
```

--- a/wm/ui/background.c
+++ b/wm/ui/background.c
@@ -253,7 +253,7 @@ static void _update(BudgieBackground *self)
                 bg_file = g_file_new_for_uri(bg_filename);
 
 #if META_MINOR_VERSION > 14
-                meta_background_set_file(background, style);
+                meta_background_set_file(background, bg_file, style);
 #else
                 char *filename = g_file_get_path(bg_file);
                 if (filename) {

